control_plane/src/compute.rs

```diff
@@ -284,7 +284,7 @@ impl PostgresNode {
         conf.append("max_wal_senders", "10");
         // wal_log_hints is mandatory when running against pageserver (see gh issue#192)
         // TODO: is it possible to check wal_log_hints at pageserver side via XLOG_PARAMETER_CHANGE?
-        conf.append("wal_log_hints", "on");
+        conf.append("wal_log_hints", "off");
```

Comment:
The comment about https://github.com/neondatabase/neon/issues/192 is obsolete, that was fixed a long time ago. Let's remove this altogether. wal_log_hints=off is the built-in default in PostgreSQL.

Reply:
I do not completely understand: do you suggest remove setting of `wal_log_hints=off` because it is Postgres default?
What about changing default value of `max_replication_write_lag` to 15Mb in the same PR?
Or it is better to create separate PR for it?